Style system ks

kstudilina opened the pull request:
Implemented styles (green border + fixed width) for the container element

kstudilina commented:
@azouaoui-deloitte  It is not exactly clear for me how to resolve the conflicts mentionned below

azouaoui-deloitte commented:
> @azouaoui-deloitte It is not exactly clear for me how to resolve the conflicts mentionned below

here is a quick explanation for the merge conflict 
https://www.gitkraken.com/learn/git/tutorials/how-to-resolve-merge-conflict-in-git

these files (on distant branche)  changed maybe while you were doing changes on your local branche?

I'm wondering if it's possible to do smaller pull requests because here we have 100 files changed :(
<img width="257" alt="Capture d’écran, le 2021-08-03 à 13 49 00" src="https://user-images.githubusercontent.com/77859905/128062280-1e225e2c-77b1-4e36-b03b-bfd2e2c9c8c1.png">


For the merge conflict we can setup a meeting to resolve them if you're not able to.

kstudilina commented:
@azouaoui-deloitte Honestly, I have no idea where these 100 files come from, I changed like 10 files max, probably some auto generated and modified things.   P.S. Thank you for the link about merge conflicts, I am going to check it and will let you know if I could resolve them myself
